membership_policy.md

```diff
@@ -128,6 +130,45 @@ The following apply to the part of project for which one would be an approver in
   * Mentor contributors and reviewers
   * May approve contributions for acceptance
 
+## SIG Chair
+
+### Requirements
+
+* contributor
```

Comment:
I'd probably want SIG chairs to have been 'Members for at least 3 months' , similar to what we have for 'Reviewer'. Maybe even better to have been 'Reviewer for at least 3 months'? Will there be a scenario where that would be an issue?

Reply:
Suggestion: sig-reviewer for at least 3 months

Reply:
resorted to another change